### ISSUE TITLE
Problem at booting cm12

### ISSUE THREAD

**Issue by iicc1** [iicc1/android_device_meizu_m1note#1]
After flashing cm12.1 I only can see Meizu logo and the phone reboots forever.
I think this problem has to be that kernel fails to boot or maybe a Selinux related problem.

I only can test builds in a M1 Note one time a week so I go very slow..

Any help would be highly appreciated

**Comment by QSCFTHMKO** [comment]
Meizu's devices may have Kernel validates in uboot.My Chinese friends do a small change in stock kernel and flash it,then boot stay at Meizu logo...

**Comment by iicc1** [comment]
They tried with unlocked bootloader I supose.

Yes but Xen0n succesufully did it with mx4, I dont know

**Comment by QSCFTHMKO** [comment]
They all work with unlocked bootloader in xda…

Maybe you can ask Xen0n for some help…

**Comment by iicc1** [comment]
Yes I have asked him a week ago. He told me to change some Selinux rules. I have compiled the build from his system-core-framework forked repo.

Now I have the build compiled but I cant test if it is solved until Friday (I dont own a m1 note)

**Comment by QSCFTHMKO** [comment]
Some of my Chinese friends are thinking about to force unlock Meizu's bootloader but I don't think they can do that…

Waiting for your good news……(ಥ⌣ಥ)

**Comment by iicc1** [comment]
I couldn’t test it  these days :(
Do you know anyone who wants to try it?
Here it is: https://mega.nz/#!G10AwQ6R!m641OX9xsq0jgCc3J4wVcPJE9sgKb2dgntGN6b59VEU

Yo wolud need to reflash stock boot.img if it doesnt works (very likely)

**Comment by QSCFTHMKO** [comment]
There are several people try this,and bootloop at Meizu logo.When see Meizu logo,then reboot at 5s later.

**Comment by iicc1** [comment]
Same as the previous build :(
Just reflash stock boot.img and stock recovery to return to flyme
